app/models/workflow_execution.rb

```diff
@@ -63,6 +63,7 @@ def sent_to_ga4gh?
 
   def as_wes_params
     {
+      name:,
```

Comment:
**nitpick (blocking):** this needs to be removed `name` is not a valid wes param.

Reply:
Removed in b6ef470f60c7206f0c965493a15ec12e6cbacb24